test/expression-to-formula.test.js

```diff
@@ -130,3 +130,20 @@ test('literal({ foo: 1, bar: 2 })', () => {
     );
   }
 });
+
+test('3 != 4', () => {
+  const actual = expressionToFormula(['!=', 3, 4]);
+  expect(actual).toEqual('3 != 4');
+});
+
+test('case(get("foo") <= 4, 6, 2 == 2, 3, 1)', () => {
+  const actual = expressionToFormula([
+    'case',
+    ['<=', ['get', 'foo'], 4],
+    6,
+    ['==', 2, 2],
+    3,
+    1
+  ]);
+  expect(actual).toEqual('case(get("foo") <= 4, 6, 2 == 2, 3, 1)');
+});
```

Comment:
Do you think it would be worth adding some tests with other in-between expressions, like math operators, to test that the decision operator is always on the outside?

Reply:
Oh god ... order of operators huh. You're right, I did not consider that.

Reply:
Wow, turned out it just worked.